resources/js/components/modules/media/CardFlip.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="media">
+    <div class="media__card col-12 col-md-6 col-lg-4" v-for="(item, index) in slide.card" :key="index">
+      <div @click="item.show=!item.show" v-bind:class="[item.show ? 'media__card--active' : '', 'media__card-wrapper pointer transition-350ms']">
+        <div class="media__card--front">
```

Comment:
`meida__card--front` & `--back` aren't really modifiers. You should give them another name. Maybe `media__card-front` and same for `back`